common/web/types/src/kvk/kvk-file.ts

```diff
@@ -5,6 +5,7 @@ import * as r from 'restructure';
 //
 
 export const BUILDER_KVK_HEADER_IDENTIFIER = 0x464B564B; // 'KVKF', little-endian
+export const KVK_HEADER_IDENTIFIER_BYTES = [0x4B, 0x56, 0x4B, 0x46];
```

Comment:
Two questions.
1. Wondering why no `BUILDER_` prefix?
2. Is this like `BUILDER_KVK_HEADER_IDENTIFIER` in big-endian? e.g. `0x4B564B46`?

Reply:
1. I left `BUILDER_` out because it's a define not used by the builder. But perhaps better to be removing a number of the `BUILDER_` prefixes in the future from other objects as well.
2. Yes, it's in array form for use in `Buffer` comparisons.